src/plugins/ovp/components/livestreams-screen/LivestreamsCard.jsx

```diff
@@ -43,6 +44,7 @@ const LivestreamsCard = ({ nodes, metas, reloadChannelState }) => nodes.map((nod
   const canViewIngests = policy.isGranted(`${APP_VENDOR}:ovp.medialive:command:stop-channel`);
   const nodeRef = NodeRef.fromNode(node).toString();
   const mediaLiveData = {};
+  useRaven({ node, nodeRef });
```

Comment:
@meghedi is this going to send a heartbeat and start collaboration for each of these videos?

Reply:
@efjacobson yes. this would add visibility to the livestreams page to show anyone editing it, similar to other node pages. let me know if you think it smells.

![CleanShot 2024-03-21 at 18 53 22](https://github.com/triniti/cms-js/assets/1748115/14beb54f-3e9b-4648-82e3-5c76b475c10a)

Reply:
@rsumilang i like that you can see who is editing it, but the person who is on the livestreams card shouldn't appear to others as if they are editing it (as the start/stop operations have no effect on the associated nodes). seems to me it's essentially like one of the search screen tables where you can see the active editors but aren't one yourself

Reply:
@efjacobson fair enough

Reply:
@rsumilang @efjacobson agree, should be like search page... just the collaborators component.